zeebe/clients/java/src/test/java/io/camunda/zeebe/client/OAuthCredentialsProviderTest.java

```diff
@@ -572,7 +575,7 @@ private void mockAuthorizedRestRequest() throws JsonProcessingException {
       wireMockInfo
           .getWireMock()
           .register(
-              WireMock.get("/v2/topology")
+              WireMock.get(URL_TOPOLOGY)
                   .withHeader("Authorization", WireMock.equalTo(TOKEN_TYPE + " " + ACCESS_TOKEN))
                   .willReturn(
                       WireMock.aResponse()
```

Comment:
❓ Looking at this, I wonder whether we can also refactor this test class to a ClientRestTest. What do you think?

Reply:
>Would it help to make the URLs in RestGatewayService public so this is the collection of known test endpoints in the client?

I like that idea a lot. Makes it easier to see coverage too